rai/node/rpc.cpp

```diff
@@ -1790,6 +1790,18 @@ class history_visitor : public rai::block_visitor
 					tree.put ("subtype", "change");
 				}
 			}
+			else if (balance == previous_balance && !handler.node.ledger.epoch_link.is_zero () && block_a.hashables.link == handler.node.ledger.epoch_link)
+			{
+				if (raw)
+				{
+					tree.put ("subtype", "epoch");
+				}
+				else
+				{
+					tree.put ("type", "epoch");
```

Comment:
I don't think we should include them in the history at all otherwise, just like with change blocks.